Generals/Code/GameEngine/Source/Common/System/StackDump.cpp

```diff
@@ -86,6 +87,12 @@ _asm
  mov eax, ebp
  mov dword ptr [myebp] , eax
 }
+#else
+	RtlCaptureContext(&gsContext);
+	myeip = gsContext.Eip;
+	myesp = gsContext.Esp;
+	myebp = gsContext.Ebp;
+#endif
```

Comment:
```
#else
#error not implemented
#endif
```

Reply:
Same as higher, it would basically hold for file as a whole, which would currently not even compile on anything other than Windows x86 (32-bit).

Reply:
That code in particular will also not compile on anything other that Windows x86, because it accesses x86 specific 32-bit registers ( Eip Esp Ebp).